src/js/modules/infragistics.util.jquery.js

```diff
@@ -122,236 +122,277 @@
 	//checkbox markup classes can be changed
 	//and setting them in a a box can be done by the predefined classes "ui-state-default ui-corner-all ui-igcheckbox-small"
 	$.ig.checkboxMarkupClasses = "";
+	$.ig.getRegionalOptions = function (reg) {
```

Comment:
My question is why `getRegionalValue` belongs to util. It's strange to me that there is a function which takes the object it extracts from as a parameter and is on a utility level.

Can you show the examples of how, and from where this method is used, apart from the formatter? Also, how is this integrated in the editors?